public/content/documentation/web/component/table.md

```diff
@@ -4,192 +4,248 @@ How to test a table
+         <tbody>
+         <tr>
+            <th scope="row">
+               <img src="/assets/images/icons/logo-apple.svg" role="img" alt="Apple" class="icon">
+               iOS
+            </th>
+            <td>
+               <img src="/assets/images/icons/logo-apple.svg" role="img" alt="Apple" class="icon">
+               VoiceOver
+            </td>
+            <td>
+               <img src="/assets/images/icons/logo-safari.svg" role="img" alt="Apple" class="icon">
+               Safari
+            </td>
+         </tr>
+         <tr>
+            <th scope="row">
+               <img src="/assets/images/icons/logo-android.svg" role="img" alt="" class="icon">
+               Android
+            </th>
+            <td>
+               <img src="/assets/images/icons/logo-talkback.svg" role="img" alt="" class="icon">
+               Talkback
+            </td>
+            <td>
+               <img src="/assets/images/icons/logo-chrome.svg" role="img" alt="" class="icon">
+               Chrome
+            </td>
+         </tr>
+         <tr>
+            <th scope="row">
+               <img src="/assets/images/icons/logo-windows.svg" role="img" alt="" class="icon">
+               Windows
+            </th>
+            <td>
+               <img src="/assets/images/icons/logo-jaws.svg" role="img" alt="" class="icon">
+               JAWS
+            </td>
+            <td>
+               <img src="/assets/images/icons/logo-chrome.svg" role="img" alt="" class="icon">
+               Chrome
+            </td>
+         </tr>
+         <tr>
+            <th scope="row">
+               <img src="/assets/images/icons/logo-windows.svg" role="img" alt="" class="icon">
+               Windows
+            </th>
+            <td>
+               <img src="/assets/images/icons/logo-nvda.svg" role="img" alt="" class="icon">
+               NVDA
+            </td>
+            <td>
+               <img src="/assets/images/icons/logo-chrome.svg" role="img" alt="" class="icon">
+               Chrome
+            </td>
+         </tr>
+         <tr>
+            <th scope="row">
+               <img src="/assets/images/icons/logo-apple.svg" role="img" alt="Apple" class="icon">
+               MacOS
+            </th>
+            <td>
+               <img src="/assets/images/icons/logo-apple.svg" role="img" alt="Apple" class="icon">
+               VoiceOver
+            </td>
+            <td>
+               <img src="/assets/images/icons/logo-safari.svg" role="img" alt="Apple" class="icon">
+               Safari
+            </td>
+            </tr>
+         </tbody>
+      </table>
+   </div>
+</example>
+
+### When you can’t use semantic HTML
+If it’s required to display tabular data without using a `<table>` element, attributes will have to added.
 
 ```html
 <div role="table" aria-describedby="table-desc">
-  <div id="table-desc">Nato phonetic alphabet</div>
+  <div id="table-desc">
+    Nato phonetic alphabet
```

Comment:
You could pull in the same Sesame Street character/letter data you used in the first example.